packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -338,11 +331,8 @@ class Transforms {
     }
 
     if (!space.encryptionKeyUrl) {
-      // The encryptionKeyUrl may not be necessary if the space name is unencrypted, so we treat it as a warning, rather than an error
-      const reason = `Space ${space.spaceId} cannot be decrypted due to a missing encryption key url`;
-
-      ctx.webex.logger.warn(reason);
-      space.warn = reason;
+      // If the encryptionKeyUrl is empty we assume the space name is unencrypted
+      ctx.webex.logger.info(`Space ${space.spaceId} cannot be decrypted due to a missing encryption key url`);
```

Comment:
Looking at the else statement below it'd be neater to bring it out as an if conditional like this one.

e.g.

    else {
      const reason = `No user available with which to decrypt space ${space.spaceId}`;
      ctx.webex.logger.error(reason);
      space.error = reason;
      return Promise.resolve(object);
    }

could become 

    if (!space.onBehalfOfUser) {.....}

That'd put all your end conditionals "up front". But this is not a requirement for acceptance.